frontend/app/components/person-cv-export.js

```diff
@@ -1,27 +1,65 @@
 import Component from "@glimmer/component";
 import { action } from "@ember/object";
 import { inject as service } from "@ember/service";
+import { isBlank } from "@ember/utils";
 
 export default class PersonCvExport extends Component {
   @service download;
   @service router;
+  @service store;
 
-  init() {
-    super.init(...arguments);
+  locations = this.store.findAll("branch_adress");
+  selectedLocation = this.locations.get("firstObject");
+
+  constructor() {
+    super(...arguments);
+  }
+
+  @action
+  handleFocus(select, e) {
+    if (this.focusComesFromOutside(e)) {
+      select.actions.open();
+    }
+  }
+
+  @action
+  handleBlur() {}
+
+  @action
+  focusComesFromOutside(e) {
+    let blurredEl = e.relatedTarget;
+    if (isBlank(blurredEl)) {
+      return false;
+    }
+    return !blurredEl.classList.contains("ember-power-select-search-input");
+  }
+
+  @action
+  setLocation(location) {
+    Ember.set(this, "selectedLocation", location);
```

Comment:
`this.selectedLocation = location`
Should be working...

Reply:
I had that before and sadly it didn't work. I'm not sure why but with Ember.set it works as expected.